### PR TITLE
Fix Cyrillic TOC

### PR DESCRIPTION
**Pull request by MizukiTemma**
### Short description

This PR tries to solve the problem that table of content in cyrillic letters is not shown in exported PDF.

Though there is an on-going discussion about PDF (library), I open this PR because this issue is probably urgent for the current situation.

### Proposed changes

- Replace the font set with a version which includes also cyrillic.
- Raleway font is used as before, but this does not support greek letters (also as before). The problem stays unsolved for greek letters but will be sloved for cyrillic. 

### Side effects

- so for not known

### Resolved issues

Fixes: #1532 

<hr>

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Review by JoeyStk — COMMENTED** [review]
Thank you so much for this PR. This is a very annoying bug, so thank you for trying to fix this. 
1. Unfortunately I still ran into the issue at one point (s. screenshot). However, this is a very strange bug. If I selected "Ласкаво просимо до міста Аугсбург" and "Відомство у справах іноземців" I ran into the issue. If I added a third page, the bug didn't appear anymore. With my current understanding this makes no sense at all to me. 
2. At least as far as my tests went, we only seem to have had this issue with Russian and Ukrainian, not with Bulgarian. This is odd to me, because Bulgarian is also an cyrillic language. However Russian and Ukrainian have a few letters more than Bulgarian, so now I'm wondering if it might have to do with these extra characters, that are unique to Ukrainian and Russian. But I would need to do some extra testing, to prove or debunk this guess.
3. Could you please add an entry to the changelog?

Thank you a lot and I hope this already helps a little bit. 
image
The table of contents for "Ласкаво просимо до міста Аугсбург" and "Відомство у справах іноземців"

**Comment by ulliholtgrave** [comment]
@JoeyStk Could you please provide a description about how we can reproduce this? I did the same and had no issue with creating two russian pages that appear in the TOC.

**Comment by JoeyStk** [comment]
I'm having this issue only in Ukrainian for specific page titles, not all of them. That's what I think is so strange. The two pages that broke the TOC were "Ласкаво просимо до міста Аугсбург" and "Відомство у справах іноземців". I copied their content from the app

**Comment by membralala** [comment]
Since both of you did not seem to have any issues with Russian language, I did not check again, but I tried to reproduce @JoeyStk 's issue with Ukrainian. For me it worked fine: 
Bildschirmfoto 2022-09-22 um 10 37 10
I also just copied the whole content blocks from the app. I tried a few more combinations, but could not break it.

**Comment by MizukiTemma** [comment]
@JoeyStk @ulliholtgrave @membralala 
Thank you for the test and report!
It's a strange bug 🤔 I'll do a further research on it.

@JoeyStk 
Entry to Changelog is done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 UNRELEASED
 ----------
+* [ [#1532](https://github.com/digitalfabrik/integreat-cms/issues/1532) ] Fix cyrillic TOC
```
Comment (timoludwig) [inline CHANGELOG.md:3]:
```suggestion

* [ [#1532](https://github.com/digitalfabrik/integreat-cms/issues/1532) ] Fix cyrillic TOC
```
The changelog needs one extra line after the "unreleased" heading and two new lines between the bullet points and the next heading.

**Review by timoludwig — APPROVED** [review]
Thanks a lot, this fixes the issue for me! :+1:

> 1. Unfortunately I still ran into the issue at one point (s. screenshot). However, this is a very strange bug. If I selected "Ласкаво просимо до міста Аугсбург" and "Відомство у справах іноземців" I ran into the issue. If I added a third page, the bug didn't appear anymore. With my current understanding this makes no sense at all to me.

@JoeyStk I can only guess what's going on here, but one possibility would be that you exported these exact two pages on the develop branch before (which caused the broken PDF to be stored in the `integreat_cms/pdf/` directory for caching) and then when you switched to this branch, maybe the PDF wasn't regenerated, but the broken version was taken from the cache...
Could you delete the `integreat_cms/pdf/` directory and try to reproduce the bug again?

File: CHANGELOG.md (hunk 1 above)
Comment (MizukiTemma) [reply CHANGELOG.md:4]: Oh I forgot this styling again 🙈 Thank you for the correction.

**Review by ulliholtgrave — CHANGES_REQUESTED** [review]
Can you squash your last commit @MizukiTemma ? I guess that this is then ready to be merged :tada:

**Comment by JoeyStk** [comment]
@timoludwig Thank you for your feedback! Yes, that seems to fix the issue! I'm sorry if I cause some unnecessary research/troubleshooting. This PR is also working for me :)

**Comment by MizukiTemma** [comment]
> Can you squash your last commit @MizukiTemma ? I guess that this is then ready to be merged 🎉 And fix the conflicts of course :D

Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 UNRELEASED
 ----------
 
+<<<<<<< HEAD
 * [ [#1653](https://github.com/digitalfabrik/integreat-cms/issues/1653) ] Create new versions even if content did not change to preserve translation
 * [ [#1450](https://github.com/digitalfabrik/integreat-cms/issues/1450) ] Add API from SUMM.AI to create easy-understable German
+=======
+>>>>>>> Update CHANGELOG.md
+* [ [#1532](https://github.com/digitalfabrik/integreat-cms/issues/1532) ] Fix cyrillic TOC
```
Comment (timoludwig) [inline CHANGELOG.md:9]:
```suggestion
* [ [#1653](https://github.com/digitalfabrik/integreat-cms/issues/1653) ] Create new versions even if content did not change to preserve translation
* [ [#1450](https://github.com/digitalfabrik/integreat-cms/issues/1450) ] Add API from SUMM.AI to create easy-understable German
* [ [#1532](https://github.com/digitalfabrik/integreat-cms/issues/1532) ] Fix PDF table of contents in cyrillic alphabets
```
Please remove the git conflict markers

**Review by timoludwig — CHANGES_REQUESTED** [review]

**Review by ulliholtgrave — APPROVED** [review]
Thank, It's fine now :+1:

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
 UNRELEASED
 ----------
 
+<<<<<<< HEAD
 * [ [#1653](https://github.com/digitalfabrik/integreat-cms/issues/1653) ] Create new versions even if content did not change to preserve translation
 * [ [#1450](https://github.com/digitalfabrik/integreat-cms/issues/1450) ] Add API from SUMM.AI to create easy-understable German
+=======
+>>>>>>> Update CHANGELOG.md
```
[the remark this answers is not in the document]
Comment (timoludwig) [reply CHANGELOG.md]: Should already be removed, maybe you have to click "refresh"? :slightly_smiling_face:

**Pull request merged**